packages/terra-base/src/Base.scss

```diff
@@ -31,3 +31,39 @@ body {
 *::after {
   box-sizing: inherit;
 }
+
+a {
+  background-color: transparent; /* Remove the gray background on active links in IE 10. */
+  color: var(--terra-link-color, #bd0fe1);
```

Comment:
I believe the hex value should be `#0065a3`, check with @neilpfeiffer to be sure.

Reply:
Updated https://github.com/cerner/terra-core/pull/1171/commits/e15a891caa9551417a9bb51b2ec4016248acf1bd